go.mod

```diff
@@ -7,8 +7,10 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20210719174558-55d730c80aa9
+	github.com/openshift/api v0.0.0-20210722202039-91e26cfb36fc
 	github.com/openshift/build-machinery-go v0.0.0-20210712174854-1bb7fd1518d3
+	github.com/openshift/cluster-ingress-operator v0.0.0-20210824175145-977aa63c247d
```

Comment:
Better to copy in the "github.com/openshift/cluster-ingress-operator/pkg/util/retryableerror" package than import it from the operator.  (Best would be to put it in a separate library, but we can consider doing that as a follow-up.)

Reply:
Done